test/browser/step_definitions/BAVAccountDetailsSteps.step.js

```diff
@@ -0,0 +1,62 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+const {
+  AccountDetailsPage,
+  BavLandingPage,
+  ConfirmDetailsPage,
+} = require("../pages");
+const { expect } = require("@playwright/test");
+
+Given("the user is on the Account Details Screen", async function () {
+  const accDetailsPage = new AccountDetailsPage(await this.page);
+  expect(await accDetailsPage.isCurrentPage()).toBeTruthy();
+});
+
+Given(
+  "the user has entered a Sort Code of {string}",
```

Comment:
Out of interest how does this work? What is `{string}` referring to?

Reply:
Allows us to pass a string into the BDD step in the feature file that can be used as a variable in our step definitions. Also means we can have 1 generic step/method for populating accountNumber/sortCode values instead of multiple.